src/Lightning.ts

```diff
@@ -718,10 +717,7 @@ export const LightningMixin = (superclass) =>
           }
 
           if (result.is_confirmed || result.is_failed) {
-            const resultPendingTrue = await Transaction.updateMany(
-              { _journal: payment._journal, pending: true },
-              { $set: { pending: false } },
-            )
+            const resultPendingTrue = await ledger.settlePayment(payment.hash)
```

Comment:
maybe `ledger.settlePayment` should not be returning a mongoose/mongodb object?

ie: the parsing of `resultPendingTrue` with `resultPendingTrue.nModified` feels inappropriate now that we have a wrapper around mongodb

Reply:
Fixed. you are right, now it just return true or false if transactions were updated or not